DB/api/db.py

```diff
@@ -9,230 +10,318 @@
 
 
 def query(parsed_url):
-	"""
```

Comment:
It also makes it difficult to find your other changes. Please separate sweeping formatting changes from code changes.